src/Utils/getUser.ts

```diff
@@ -0,0 +1,22 @@
+export function getUser(user: User): User {
+  let currentUser = user
+
+  if (currentUser === undefined) {
```

Comment:
Seeing as this is valid, you should type the function parameter appropriately (as optional).

Reply:
Nit: It’s more idiomatic for parameters to use `?` to indicate optionality. E.g.

```ts
export function getUser(user?: User | null): User | null {
 // …
}
```

Btw, is it expected that when passing in `null` as `user` you get back `null`? Or was the intent to have `currentUser === undefined || currentUser === null` ?